Test failure: xray 0.6.0 on i686

`test_reduce` in `xray.test.test_variable.TestVariable` fails for i686 with all Python versions.

```
FAIL: test_reduce (xray.test.test_variable.TestVariable)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/nix-build-python3.4-xray-0.6.0.drv-0/xray-0.6.0/xray/test/test_variable.py", line 707, in test_reduce
    self.assertVariableIdentical(v.mean('x'), v.reduce(np.mean, 'x'))
  File "/tmp/nix-build-python3.4-xray-0.6.0.drv-0/xray-0.6.0/xray/test/__init__.py", line 122, in assertVariableIdentical
    assert as_variable(v1).identical(v2), (v1, v2)
AssertionError: (<xray.Variable (y: 3)>
array([ 0.48885496,  0.57610051,  0.33430722]), <xray.Variable (y: 3)>
array([ 0.48885496,  0.57610051,  0.33430722]))
```

The build log for Python 3.3: http://hydra.nixos.org/build/25641209/nixlog/1/raw
The build logs for other Python versions can be found at: http://hydra.nixos.org/eval/1221294?filter=xray&compare=1221225&full=1

--- a/xray/test/test_variable.py
+++ b/xray/test/test_variable.py
@@ -730,7 +730,7 @@ class TestVariable(TestCase, VariableSubclassTestCases):
         self.assertVariableIdentical(
             v.reduce(np.mean, 'x').reduce(np.std, 'y'),
             Variable([], self.d.mean(axis=0).std()))
-        self.assertVariableIdentical(v.mean('x'), v.reduce(np.mean, 'x'))
+        self.assertVariableAllClose(v.mean('x'), v.reduce(np.mean, 'x'))
 
         with self.assertRaisesRegexp(ValueError, 'cannot supply both'):
             v.mean(dim='x', axis=0)

